frontend/src/__tests__/cypress/cypress/e2e/Resources/Resources.cy.ts

```diff
@@ -0,0 +1,377 @@
+import { mockDashboardConfig, mockDscStatus, mockStatus } from '~/__mocks__';
+import { mockDocs } from '~/__mocks__/mockDocs';
+import { mockComponents } from '~/__mocks__/mockComponents';
+import { mockQuickStarts } from '~/__mocks__/mockQuickStarts';
+import { resources } from '~/__tests__/cypress/cypress/pages/resources';
+
+const initIntercepts = () => {
+  cy.intercept('/api/status', mockStatus());
+  cy.intercept('/api/dsc/status', mockDscStatus({}));
+  cy.intercept('/api/config', mockDashboardConfig({}));
+  cy.intercept('/api/docs', mockDocs());
+  cy.intercept('/api/components', mockComponents());
+  cy.intercept('/api/quickstarts', mockQuickStarts());
```

Comment:
Use new APIs in `odh.ts` and add anything additional APIs for missing endpoints.